pywbem/_subscription_manager.py

```diff
@@ -343,30 +343,38 @@ def add_server(self, server):
         self._owned_subscriptions[server_id] = []
         self._owned_filters[server_id] = []
         self._owned_destinations[server_id] = []
+        # self._systemnames[server_id] = None
```

Comment:
If the `self._systemnames` attribute is changed to a string, there is no longer a need to create an item in that dict.

Reply:
DONE